src/Presentation/Pagination/Types/TPPagination.cs

```diff
@@ -0,0 +1,20 @@
+namespace Presentation.Pagination.Types
+{
+    using System.Diagnostics.CodeAnalysis;
+    using Business.Pagination;
+    using global::GraphQL.Types;
+    using Style;
+
+    [SuppressMessage(Category.Default, Check.CA1724, Justification = Reason.Readability)]
+    public class TPPagination : InputObjectGraphType<PPagination>
+    {
+        public TPPagination()
+        {
+            Name = "Pagination";
+            Description = "It will be used for fetching data for a single page";
```

Comment:
Should we use a better naming for Name and Description like `Name = "PaginationParameter"` and `Description = "The parameter used to specify the pagination info"`?